code/controllers/subsystem/job.dm

```diff
@@ -582,6 +583,11 @@ SUBSYSTEM_DEF(job)
 			J.total_positions = officer_positions
 			J.spawn_positions = officer_positions
 
+	var/datum/job/phy = SSjob.GetJob("Brig Physician")
+	if(J.total_positions > 1)
+		phy.total_positions = 1
+		phy.spawn_positions = 1
```

Comment:
Why is this here? It just unnecessarily makes it impossible to have more than 1 brig phys, which currently won't happen outside of bus.